reactive/archive_auth_mirror.py

```diff
@@ -58,7 +58,17 @@ def config_not_set():
         'blocked', 'Not all required configs set, mirroring is disabled')
 
 
-def configure_static_serve():
+@when('leadership.is_leader')
+def install_cron():
+    cron.install_crontab()
+
+
+@when_not('leadership.is_leader')
+def remove_cron():
+    cron.remove_crontab()
```

Comment:
Good point. when I first tested it it seemed that the hook would only fire when leadership changed, but I verified it does get run every time.

Fixed by adding a new state.